tensorflow/python/keras/activations.py

```diff
@@ -170,6 +174,17 @@ def tanh(x):
 
 @keras_export('keras.activations.sigmoid')
 def sigmoid(x):
+  """Sigmoid.
+  Applies the sigmoid activation function. The sigmoid function is defined as 
+  1 divided by (1 + exp(-x)). It's curve is like an "S" and is like a smoothed 
+  version of the Heaviside function. For small values (<-5) the sigmoid returns 
```

Comment:
Maybe put "Unit Step Function" in parenthesis for the Heaviside, the whole point after all is simplicity.